.github/workflows/ci.yml

```diff
@@ -11,18 +11,29 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        shard_file:
+          - shard.yml
         crystal_version:
           - 0.36.1
           - 1.0.0
         experimental:
           - false
+        include:
+          - shard_file: shard.edge.yml
```

Comment:
Is this a new file Crystal uses?

Reply:
No, I just didn't want to create a `shard.override.yml` since that would *always* be taken by default. I named it according to how Brian C. recommended here: https://crystal-lang.org/2020/08/20/preparing-our-shards-for-crystal-1.0.html